test/common/fit_common.py

```diff
@@ -1,11 +1,11 @@
 '''
-Copyright 2016, EMC, Inc.
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
Agree.  Technically, the copyright does not need to be changed.  But if we are changing it, it should include the original date.